### PR TITLE
[SDK] Wine sync _mbtowc_l code

### PR DESCRIPTION
**Pull request by Fraizeraust**
[CORE-16885](https://jira.reactos.org/browse/CORE-16885)

### CODE REVIEW

**Comment by Fraizeraust** [comment]
**AFTER**
Capture

File: dll/win32/ieframe/ieframe.rc [hunk 1]
```diff
@@ -85,6 +85,10 @@ LANGUAGE LANG_NEUTRAL, SUBLANG_NEUTRAL
 
 #include "wine/wine_common_ver.rc"
```
Comment (learn-more) [inline dll/win32/ieframe/ieframe.rc:86]: this already includes version resources.

**Comment by HBelusca** [comment]
How does the "before" used to look like?

**Comment by Fraizeraust** [comment]
This is how it looked before (taken from the aforementioned ticket).
Sans titre

**Comment by HBelusca** [comment]
That is indeed strange because wine_common_ver.rc file contains an include of our reactos/version.rc as well, see the last line of https://git.reactos.org/?p=reactos.git;a=blob;f=sdk/include/reactos/wine/wine_common_ver.rc;h=e24debc005a01fe88c9e3a8bf6de3a9eeccc8ccd;hb=HEAD .

**Comment by HBelusca** [comment]
For completeness, could you also check whether wctomb/_wctomb_l , mbstowcs/_mbstowcs_l and wcstombs/_wcstombs_l need as well some syncings? (If so please open a separate PR.)

**Comment by Fraizeraust** [comment]
`wcsrtombs_l` (which is used by used by _wcstombs_l and as such, wcstombs), wctomb and _mbstowcs_l need a sync respectively.

**Comment by Saibamen** [comment]
Please mark this Pull Request as Draft if is still WIP: https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/changing-the-stage-of-a-pull-request

**Comment by Fraizeraust** [comment]
@Saibamen The pull request is no longer WIP but as you're aware, I'm not into the development team so I can't change the tags. I know of this Draft thing so you don't have to tell me that. 😛

**Comment by Saibamen** [comment]
@Fraizeraust That is why using Draft feature is better than labels -> both PR authors and members of dev team can change PR to Draft/Ready for Review ;)

**Pull request merged**